### ISSUE TITLE
Deploy imagebuilding to Openscapes hub

### ISSUE THREAD

**Issue by yuvipanda** [2i2c-org/infrastructure#5303]
I presented the demo to the openscapes community in one of their hackathon calls, and they would like it to be turned on for the openscapes hub.

Docs: https://infrastructure.2i2c.org/howto/features/imagebuilding/

## Definition of done

- [x] imagebuilding feature is turned on for the openscapes hub

**Comment by jmunroe** [comment]
WIP: https://github.com/2i2c-org/infrastructure/tree/openscapes-imagebuilding

I've successfully deployed binderhub-service to the openscapes hub and set up an quay.io organization as an image repository.

Currently, it is deployed on the staging hub: https://staging.openscapes.2i2c.cloud/

I've put notice in the #cloud-infrastructure channel on the Openscapes slack that I will roll this out to production on Friday assuming I receive no objections.

**Comment by GeorgianaElena** [comment]
Great work @jmunroe! 🚀

I wanted to point this issue https://github.com/2i2c-org/infrastructure/issues/5306 that I noticed on the board, because I think this is work we might pick up in the next sprint. 
If you want and can, (definitely not a blocker for this work) I figured that as part enabling imagebuilding to more hubs like openscapes, we would do so under a single, generic quay org instead of a community specific one.  Then, we can migrate the old ones as part of https://github.com/2i2c-org/infrastructure/issues/5306 and this way it will be less migration-related work. What do you think?

**Comment by jmunroe** [comment]
I agree that we should complete #5306 before deploying any new imagebuilding hubs but let's get this issue closed out and pick that task up in a future iteration.

**Comment by jmunroe** [comment]
Deployed.
